lambda_src/drivers/process_https.py

```diff
@@ -54,6 +55,7 @@ def process_row(
             else {}
         ).items()
     }
+    req_data: Optional[bytes]=None
```

Comment:
Formatting please. you need to leave spaces before variables and operators and values.